heat/core/logical.py

```diff
@@ -206,4 +208,6 @@ def any(x, axis=None, out=None, keepdim=False):
     def local_any(t, *args, **kwargs):
         return torch.any(t != 0, *args, **kwargs)
 
-    return operations.__reduce_op(x, local_any, MPI.LOR, axis=axis, out=out, keepdim=keepdim)
+    return operations.__reduce_op(
+        x, local_any, MPI.LOR, axis=axis, out=out, neutral=1, keepdim=keepdim
```

Comment:
For logical or I think the neutral value should also be 0?

Reply:
Correct, thanks!